plugin.video.fullycharged/addon.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.fullycharged"
+	name="Fully Charged"
+	version="1.0.1"
+	provider-name="zag">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>
```

Comment:
Further to @phil65 's comment. You need to set xbmc.python to the correct minimum branch (see http://kodi.wiki/view/Addon.xml#Dependency_versions for detail).
I would move the icon.png and fanart.jpg files into the addon root directory as well.